examples/trip_parser/from_base64/trip_parser.py

```diff
@@ -0,0 +1,33 @@
+
+# Starts the parsing process
+try:
+    response = amadeus.travel.trip_parser_jobs.post(
+        amadeus.travel.from_base64(booking_base64))
+    id = response.data['id']
+except ResponseError as error:
+    raise error
+
+
+# Retrieves the status of the process
+def get_status(id):
+    try:
+        response = amadeus.travel.trip_parser_jobs.status(id).get()
+    except ResponseError as error:
+        raise error
+    return response.data['status']
+
+
+# Retrieves the parsing result when the process is complete
+while get_status(id) != 'COMPLETED':
+    time.sleep(5)
```

Comment:
In the example maybe we should put a higher sleep? So people don't flood the API